pyanaconda/ui/tui/spokes/language_support.py

```diff
@@ -82,48 +92,48 @@ def refresh(self, args=None):
         NormalTUISpoke.refresh(self, args)
 
         if args:
-            self._window += [TextWidget(_("Available locales"))]
+            self.window.add(TextWidget(_("Available locales")))
             displayed = [TextWidget(localization.get_english_name(z)) for z in args]
```

Comment:
We don't need to create the `displayed` list, the items can be added to the container with a proper callback here.